### PR TITLE
BubbleSort

### PR DESCRIPTION
**Pull request by Diaboliq**
Given an array of integers, implement the BubbleSort algorithm for sorting the elements of the array

### CODE REVIEW

File: BubbleSort [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+        for (int i=0; i<array.length; i++) {
+            for (int j=0; j<array.length-1; j++) {
+                if (array[j] > array[j+1]) {
+                    int temporaryInt = array[j];
+                    array[j] = array[j+1];
+                    array[j+1] = temporaryInt;
+                }
+            }
+            //System.out.println("Iteration nr: " + i + " " + Arrays.toString(array));
```
Comment (CristianManic) [inline BubbleSort:18]: don't leave commented lines of code

File: BubbleSort [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+        int[] myArray = new int[10];
+
+        Random rand = new Random();
+        int upperLimit = 1000;
+
+        for (int i=0; i<myArray.length; i++) {
+            int int_random = rand.nextInt(upperLimit);
+            myArray[i] = int_random;
+        }
```
Comment (CristianManic) [inline BubbleSort:33]: same as for previous PR, please extract population of array into separate method

File: BubbleSort [hunk 3]
```diff
@@ -0,0 +1,43 @@
+package work;
+
+import java.util.Arrays;
+import java.util.Random;
+
+public class BubbleSort {
+
+    public static void bubbleSort(int[] array) {
+
```
Comment (CristianManic) [inline BubbleSort:9]: no need for this blank line

File: BubbleSort (hunk 1 above)
Comment (Diaboliq) [reply BubbleSort:18]: Done!

File: BubbleSort (hunk 3 above)
Comment (Diaboliq) [reply BubbleSort:9]: Ok

File: BubbleSort (hunk 2 above)
Comment (Diaboliq) [reply BubbleSort:29]: Done.

**Review by CristianManic — APPROVED** [review]

**Pull request merged**